tangelo/toolboxes/molecular_computation/molecule.py

```diff
@@ -65,6 +65,40 @@ def molecule_to_secondquantizedmolecule(mol, basis_set="sto-3g", frozen_orbitals
     return converted_mol
 
 
+def spatial_from_spinorb(one_body_coefficients, two_body_coefficients):
+    """Function to reverse openfermion.chem.molecular_data.spinorb_from_spatial.
+
+    Args:
+        one_body_coefficients: One-body coefficients (array of 2N*2N, where N
+            is the number of molecular orbitals).
+        two_body_coefficients: Two-body coefficients (array of 2N*2N*2N*2N,
+            where N is the number of molecular orbitals).
+
+    Returns:
+        (array of floats, array of floats): One- and two-body integrals (arrays
+            of N*N and N*N*N*N elements, where N is the number of molecular
+            orbitals.
+    """
+    # Get the number of MOs = number of SOs / 2.
+    n_mos = int(one_body_coefficients.shape[0] / 2)
```

Comment:
Use the `//` operator for division with integers, and then no need to cast to int.